examples/etherpad/Nulecule.yaml

```diff
@@ -0,0 +1,104 @@
+# Using etherpad as an example
```

Comment:
Why did you drop having a single definition, something like compose, for a multi container application?